src/firebase.js

```diff
@@ -15,13 +13,9 @@ const firebaseConfig = {
   measurementId: 'G-80PYR1XPLM',
 };
```

Comment:
Config musimy pilnie przenieść do plików .env, nie powinny one nigdy trafić do repozytorium.
Plik .env powinien być dodany do .gitignore! Pisz priv jeśli nie wiesz jak to ogarnąć